ophyd/controls/area_detector.py

```diff
@@ -196,6 +196,9 @@ def configure(self, state=None):
 
         # If using the stats, configure the proc plugin
 
+        # Enable images to be saved
+        self._write_plugin('EnableCallbacks', 1, self._file_plugin)
```

Comment:
On the other PR I left the comment "This should move down to the *Filestore child class"